src/account/operations/transaction/high_level/create_alias.rs

```diff
@@ -0,0 +1,169 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use iota_client::{
+    api::PreparedTransactionData,
+    block::{
+        address::Address,
+        output::{
+            feature::{Feature, MetadataFeature},
+            unlock_condition::{
+                GovernorAddressUnlockCondition, StateControllerAddressUnlockCondition, UnlockCondition,
+            },
+            AliasId, AliasOutputBuilder, Output,
+        },
+        DtoError,
+    },
+};
+use serde::{Deserialize, Serialize};
+
+use crate::{
+    account::{handle::AccountHandle, types::Transaction, OutputData, TransactionOptions},
+    Error,
+};
+
+/// Address and alias for `create_alias_output()`
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct AliasOutputOptions {
```

Comment:
Would we also be interested in `state_metadata`?

Reply:
Yes good catch